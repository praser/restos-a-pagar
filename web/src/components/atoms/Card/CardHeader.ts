import styled from 'styled-components';

const CardHeader = styled.div`
  border-radius: calc(0.35rem - 1px) calc(0.35rem - 1px) 0 0;
  padding-bottom: 1rem;
  padding-top: 1rem;
  padding: 0.75rem 1.25rem;
  margin-bottom: 0;
  background-color: #f8f9fc;
  border-bottom: 1px solid #e3e6f0;
  color: #5a5c69;
  font-weight: 700;
`;

export default CardHeader;
