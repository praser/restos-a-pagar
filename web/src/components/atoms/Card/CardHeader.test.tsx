import React from 'react';

import renderer from 'react-test-renderer';

import CardHeader from './CardHeader';

describe('CardHeader atom component', () => {
  it('is expected to render the component correctly', () => {
    const cardHeader = renderer.create(<CardHeader />).toJSON();
    expect(cardHeader).toMatchSnapshot();
  });
});
