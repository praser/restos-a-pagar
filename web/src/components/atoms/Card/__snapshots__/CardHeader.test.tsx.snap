// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader atom component is expected to render the component correctly 1`] = `
<div
  className="sc-bdfBwQ hVynvs"
/>
`;
